### PR TITLE
Can 'Sign In', 'Sign Up', and get college list

### PR DESCRIPTION
**Pull request by Tommybro**
Now, frontend communicates with backend.
So, only valid user(student) can sign in.
And frontend gets college, department and major lists from backend.
And user can sign up.
In sign up page, frontend get necessary informations from the user.
In sign up page, user can go back to sign in page.

### CODE REVIEW

**Comment by holenet** [comment]
Please make title to descriptive for the content of implementaions.

File: src/components/molecules/SignUp/index.js [hunk 1]
```diff
@@ -1,18 +1,32 @@
-import React, { PropTypes } from 'react'
```
Comment (holenet) [inline src/components/molecules/SignUp/index.js]: Why did you delete `PropTypes`?

**Comment by Tommybro** [comment]
Is there any other necessary modification?

File: src/components/molecules/SignUp/index.js (hunk 1 above)
Comment (Tommybro) [reply src/components/molecules/SignUp/index.js]: `PropTypes` tells whether given prop's type is same with expected prop's type through warning.
It's purpose is good.
But whenever I use array, I got not understandable warning (Although it works very well as expected).
So I removed `PropTypes` not to get useless(I think) warning.

Comment (holenet) [reply src/components/molecules/SignUp/index.js]: As you mentioned, even if you do not use `PropTypes`, overall implementations would not get big problem.
please continue development in the present way.

Comment (Tommybro) [reply src/components/molecules/SignUp/index.js]: OK
